lib/Core/Service/InformationCollectionService.php

```diff
@@ -191,7 +190,14 @@ public function getCollections(ContentId $contentId): Collections
 
     public function filterCollections(FilterCriteria $criteria): Collections
     {
-        // TODO: Implement filterCollections() method.
+        $collections = $this->ezInfoCollectionRepository->filterByIntervalOfCreation($criteria->getContentId()->getContentId(), $criteria->getFrom(), $criteria->getTo());
```

Comment:
It does not seem that this is generic `filter` method now. Maybe better to introduce new method `filterByPeriod` or something, and leave this empty with a TODO?